<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetAccountWizardPage</class>
 <widget class="QWizardPage" name="AssetAccountWizardPage">
  <attribute name="title">
    <string>Asset Account</string>
  </attribute>
  <layout class="QVBoxLayout">
    <item>
      <widget class="QLabel" name="TextLabel10_4_2">
        <property name="text">
          <string>If this loan is for an asset, such as a car or a house, you can create the asset account now. An asset account represents the total value of an asset. The money from this loan will be transferred into the asset account you create or select.
If this loan is a 'consumer loan' (money to use however you want), you can use a checking account instead.</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <spacer name="spacer80">
        <property name="sizeHint">
          <size>
            <width>20</width>
            <height>21</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <widget class="QCheckBox" name="m_dontCreatePayoutCheckBox">
        <property name="text">
          <string>Do not create payout transaction</string>
        </property>
      </widget>
    </item>
    <item>
      <layout class="QVBoxLayout">
        <item>
          <widget class="kMyMoneyAccountSelector" name="m_assetAccountEdit"/>
        </item>
        <item>
          <layout class="QHBoxLayout">
            <item>
              <spacer name="spacer244">
                <property name="sizeHint">
                  <size>
                    <width>466</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
              </spacer>
            </item>
            <item>
              <widget class="QPushButton" name="m_createNewAssetButton">
                <property name="text">
                  <string/>
                </property>
                <property name="shortcut">
                  <string/>
                </property>
                <property name="toolTip" stdset="0">
                  <string>Press this to create a new asset account</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>
      </layout>
    </item>
    <item>
      <layout class="QHBoxLayout">
        <item>
          <widget class="QLabel" name="textLabel2">
            <property name="text">
              <string>Date of payment</string>
            </property>
            <property name="wordWrap">
              <bool>false</bool>
            </property>
          </widget>
        </item>
        <item>
          <widget class="kMyMoneyDateInput" name="m_paymentDate"/>
        </item>
      </layout>
    </item>
    <item>
      <spacer name="spacer328">
        <property name="sizeHint">
          <size>
            <width>20</width>
            <height>16</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
 </customwidgets>
 <tabstops>
 </tabstops>
</ui>
